core/src/main/scala/kafka/server/KafkaConfig.scala

```diff
@@ -74,6 +74,7 @@ object Defaults {
   val InitialBrokerRegistrationTimeoutMs = 60000
   val BrokerHeartbeatIntervalMs = 2000
   val BrokerSessionTimeoutMs = 9000
+  val MetadataSnapshotMaxNewRecordBytes = Long.MaxValue
```

Comment:
Can you please set a sensible default here so that snapshots are not disabled by default?

Reply:
Okay. Set it to 20MB. This is the value documented in KIP-630.